documentation/modules/adding-source-provider.adoc

```diff
@@ -15,7 +15,7 @@ EMS enforcement is disabled for migrations with VMware vSphere source providers
 
 .Prerequisites
 
-* It is strongly recommended to create a VMware Virtual Disk Development Kit (VDDK) image in a secure registry that is accessible to all clusters. A VDDK image accelerates migration and reduces the risk of a plan failing. If you are not using VDDK and a plan fails, then please retry with VDDK installed. For more information, see xref:../master.adoc#creating-vddk-image_mtv[Creating a VDDK image].
+* It is strongly recommended to create a VMware Virtual Disk Development Kit (VDDK) image in a secure registry that is accessible to all clusters. A VDDK image accelerates migration and reduces the risk of a plan failing. If you are not using VDDK and a plan fails, then please retry with VDDK installed. For more information, see xref:../master.adoc#creating-vddk-image_vmware[Creating a VDDK image].
```

Comment:
Since this is a pre-req, maybe we can dispense with "It is strongly recommended."  We don't seem to tell them what happens if they don't follow our recommendations.  We don't say who is making the recommendation either.  I know we can't say "Red Hat." Maybe start with, "Create a VMware Virtual Disk..."  

I'd delete "please retry"; we never use please in tech doc.

Reply:
@RichardHoch Make this change in the PR for creating providers.